dvbtee/dvbtee.cpp

```diff
@@ -359,10 +364,11 @@ int main(int argc, char **argv)
 
 	int num_tuners           = -1;
 	unsigned int timeout     = 0;
+        
+        unsigned int server_port = SERVE_DEFAULT_PORT;
```

Comment:
leading spaces rather than a tab...   Since the rest of this project uses tabs, let's just stay consistent